after/plugin/dracula.vim

```diff
@@ -2,6 +2,12 @@ if dracula#should_abort()
   finish
 endif
 
+function! s:GitSignsInstalled()
+  " https://github.com/lewis6991/gitsigns.nvim requires nvim > 0.5
+  " has('nvim-0.5') checks >= 0.5, so this should be future-proof.
+  return has('nvim-0.5') && luaeval("pcall(require, 'gitsigns')")
```

Comment:
I feel like this should just be inlined on line 56.

Reply:
@benknoble Don't have strong feelings on this, just a suggestion. I'm cool with you merging whenever you're ready. The reins are all you on this one.

Reply:
I agree w/ the inlining. Changed the code.